web/views.py

```
@@ -2382,14 +2398,14 @@ def teacher_dashboard(request):
     return render(request, "dashboard/teacher.html", context)
 
 
-def custom_404(request, exception):
-    """Custom 404 error handler"""
-    return render(request, "404.html", status=404)
+def custom_404(request, exception=None):
```

Ruff (0.8.2)

Missing return type annotation for public function `custom_404`

(ANN201)